server/jetstream_cluster.go

```diff
@@ -8048,6 +8048,18 @@ func (mset *stream) processClusteredInboundMsg(subject, reply string, hdr, msg [
 			mset.storeMsgIdLocked(&ddentry{msgId, 0, time.Now().UnixNano()})
 			mset.mu.Unlock()
 		}
+
+		// TTL'd messages are rejected entirely if TTLs are not enabled on the stream.
+		if ttl, _ := getMessageTTL(hdr); ttl != 0 && !allowTTL {
+			if canRespond {
+				var resp = &JSPubAckResponse{PubAck: &PubAck{Stream: name}}
+				resp.PubAck = &PubAck{Stream: name}
```

Comment:
You set this above already. I think you can remove this line.

Reply:
Oops, yes. Fixed.